hcloud/action_test.go

```diff
@@ -74,6 +74,34 @@ func TestActionClientList(t *testing.T) {
 		if perPage := r.URL.Query().Get("per_page"); perPage != "50" {
 			t.Errorf("expected per_page 50; got %q", perPage)
 		}
+
+		r.ParseForm()
+		status := r.Form["status"]
```

Comment:
You can access query string parameters easier:

```go
r.URL.Query().Get("status")
```

Reply:
This does not work with parameters which can be specified multiple times:
https://play.golang.org/p/N40PisXfxa9

Reply:
Oh, I see.

> To access multiple values, use the map directly.

https://golang.org/pkg/net/url/#Values.Get